let User,CustomErrorHandler;_4b4‍.x([["default",()=>_4b4‍.o]]);_4b4‍.w("../../models",[["User",["User"],function(v){User=v}]]);_4b4‍.w("../../services/CustomErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userDetailsControllers = {
  async singleUserData(req, res, next) {
    try {
      const user = await User.findOne({ _id: req.user._id }).select(
        "-password -updatedAt -__v"
      );
      if (!user) {
        return next(CustomErrorHandler.notFound());
      }
      res.json(user);
    } catch (err) {
      return next(err);
    }
  },
};

_4b4‍.d(userDetailsControllers);
